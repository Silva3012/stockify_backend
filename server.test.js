require('dotenv').config();
const request = require('supertest');
const jwt = require('jsonwebtoken');
const app = require('./server');

// Snapshot
// Expect the exported object from server.js to match the snapshot
test('server.js matches the snapshot', () => {
  expect(app).toMatchSnapshot();
});

// Unit test 
describe('Server', () => {
    it('should have the expected properties and methods', () => {
        // Test if the app object has the expected properties and methods
        expect(app).toHaveProperty('use');
        expect(app).toHaveProperty('listen');
    });

    it('should listen on the specified port', () => {
        // Expected port the server should listen on
        const expectedPort = 3001;
    }); 
}); 