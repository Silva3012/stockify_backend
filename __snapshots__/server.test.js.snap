// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`server.js matches the snapshot 1`] = `[Function]`;
